src/main/java/com/greenfox/poker/model/GamePlayer.java

```diff
@@ -0,0 +1,52 @@
+package com.greenfox.poker.model;
+
+
+public class GamePlayer {
+
+  private Action lastAction;
```

Comment:
`lastAction ` should be a String, not an object. Pay more attention to the specification!